cookbook-react/src/recipes/lists/list-cmpc/styles.module.scss

```diff
@@ -0,0 +1,58 @@
+$alabaster-tertiary: #FCFCFC;
+$alabaster: #F7F7F7;
+$oslo-gray: #92978B;
+$black: #000;
+
+.list-item {
+  border-radius: 6px;
+  height: 50px;
+  padding: 20px 20px 20px 25px;
+  position: relative;
+
+  &:nth-child(2n) {
+    background-color: $alabaster-tertiary;
+  }
+
+  &:nth-child(2n - 1) {
+    background-color: $alabaster;
+  }
+
+  &:not(:last-child) {
+    margin-bottom: 12px;
+  }
+}
+
+.text-normal {
+  color: $black;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 1.2;
+}
+
+.close-icon {
```

Comment:
In the image it seems like the close icon is not centered vertically, please check that